chain/digibyte/digibyte_test.go

```diff
@@ -0,0 +1,117 @@
+package digibyte_test
+
+import (
+	"context"
+	"log"
+	"os"
+	"reflect"
+	"time"
+
+	"github.com/btcsuite/btcutil"
+	"github.com/renproject/id"
+	"github.com/renproject/multichain/chain/digibyte"
+	"github.com/renproject/multichain/compat/bitcoincompat"
+	"github.com/renproject/pack"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("DigiByte", func() {
+	Context("when submitting transactions", func() {
+		Context("when sending DigiByte to multiple addresses", func() {
+			It("should work", func() {
+				// Load private key, and assume that the associated address has
+				// funds to spend. You can do this by setting DIGIBYTE_PK to the
+				// value specified in the `./multichaindeploy/.env` file.
+				pkEnv := os.Getenv("DIGIBYTE_PK")
+				if pkEnv == "" {
+					panic("DIGIBYTE_PK is undefined")
+				}
+				wif, err := btcutil.DecodeWIF(pkEnv)
+				Expect(err).ToNot(HaveOccurred())
+
+				// PKH
+				pkhAddr, err := btcutil.NewAddressPubKeyHash(btcutil.Hash160(wif.PrivKey.PubKey().SerializeCompressed()), digibyte.DigiByteRegtestParams)
+				Expect(err).ToNot(HaveOccurred())
+				pkhAddrUncompressed, err := btcutil.NewAddressPubKeyHash(btcutil.Hash160(wif.PrivKey.PubKey().SerializeUncompressed()), digibyte.DigiByteRegtestParams)
+				Expect(err).ToNot(HaveOccurred())
+				log.Printf("PKH                %v", pkhAddr.EncodeAddress())
+				log.Printf("PKH (uncompressed) %v", pkhAddrUncompressed.EncodeAddress())
+
+				// Setup the client and load the unspent transaction outputs.
+				client := bitcoincompat.NewClient(bitcoincompat.DefaultClientOptions().WithHost("http://127.0.0.1:18443"))
```

Comment:
To match the suggestion of changing DigiByte port to `20443` (Bitcoin has already reserved `18443`):
```suggestion
				client := bitcoincompat.NewClient(bitcoincompat.DefaultClientOptions().WithHost("http://127.0.0.1:20443"))
```